stdlib/runtime/numbers.gr

```diff
@@ -1402,3 +1418,19 @@ export let decr = (x) => {
   @deprecated Please use `Numbers.decr` instead. `Numbers.numberDecr` will be removed in 0.4.0.
 */
 export let numberDecr = decr;
+
+export let min = (x, y) => {
+  if (x < y) {
+    x
+  } else {
+    y
+  }
+}
+
+export let max = (x, y) => {
+  if (x < y) {
+    y
+  } else {
+    x
+  }
+}
```

Comment:
These are now in the `Number` library, so I think we should rely on those instead of runtime/numbers.